apps/web/src/components/pricing/standardPricingCard.tsx

```diff
@@ -25,7 +25,7 @@ const featureList = [
     title: "Scheduled Video Uploads",
     description: "Automatically post videos to YouTube.",
   },
-] as const;
+];
```

Comment:
Why you deleted this?

Reply:
Propably overlooked it while rresolving merge conflicts.